examples/charm++/load_balancing/kNeighbor/kNeighbor.C

```diff
@@ -238,6 +305,13 @@ class Block: public CBase_Block {
       p(curIterMsgSize);
       p(internalStepCnt);
       p(sum);
+      p(msg_var);
+      p(compute_var);
+      p(compute_work);
```

Comment:
Minor, but it'd be good to maintain the same ordering as these are declared and initialized in (`compute_work` and then `compute_var`).